CMakeLists.txt

```diff
@@ -6,32 +6,36 @@ set(CMAKE_BUILD_TYPE Release)
 #set(CMAKE_CXX_COMPILER /usr/local/bin/g++-4.8)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/misc/")
-find_package(LibXml2 REQUIRED)
```

Comment:
Right.. but when you use the distribution-provided libxml e.g. on Linux, you'll get a linker error in this case.